client/src/components/new-request/main-container.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <v-container fluid class="white rounded copy-normal">
+  <v-container fluid class="white rounded copy-normal pa-10">
```

Comment:
This is the gutter (whitespace) around the main container (which is used for most or all pages). And all content should be tight up to this.